base_classes/NXtransformations.nxdl.xml

```diff
@@ -325,7 +325,8 @@
 		<attribute name="depends_on" type="NX_CHAR">
 			<doc>
 				Points to the path to a field defining the axis on which this
-				depends or the string "." when at the end of the chain.
+				depends or the string ".". It can also point to an instance of
+				``NX_coordinate_system`` if this transformation depends on it.
```

Comment:
To understand this last sentence, you would like to offer the possibility to define explicitly in which coordinate system the transformation is living right instead of leaving it implicit in that there is e.g. no parent of that instance being an NXcoordinate_system or just constructs like NXcoordinate_system with some NXtranslations in it.
If this understanding is correct I fully support

Reply:
Yup, that's right